### DISCUSSION TITLE
Patreon link

### DISCUSSION THREAD

**Discussion by Xukkorz** [community/community#31130] [Other Feature Feedback, Questions, & Ideas] [answered]
Your patreon link is still the old one on this profile fyi

**Comment by airtower-luna** [comment, marked as answer]
That message looks like it should've gone to a specific person, but this is a public forum, not a personal message. :sweat_smile: Try reaching out to the person by mail or something else, if you can.

**Reply by Xukkorz** [reply]
Yeah well I thought that's what I did, I got to it from a specific person's git, thanks for letting me know that didn't happen...
